frontend/javascripts/oxalis/view/action-bar/tracing_actions_view.js

```diff
@@ -172,10 +179,24 @@ class TracingActionsView extends React.PureComponent<Props, State> {
     isShareModalOpen: false,
     isMergeModalOpen: false,
     isUserScriptsModalOpen: false,
+    isReopenAllowed: false,
   };
 
   modalWrapper: ?HTMLDivElement = null;
 
+  componentDidUpdate = () => {
+    const localStorageEntry = UserLocalStorage.getItem("lastFinishedTask");
+    if (this.props.task && localStorageEntry) {
+      const { finishedTime } = JSON.parse(localStorageEntry);
+      if (Date.now() - finishedTime < 30000) {
```

Comment:
Ideally, the 30 seconds should be imported from the WK config. If it's too complicated, simply define it in a constant (`constants.js`) and don't expose it in the wk config. @daniel-wer Do you know whether we have a way to read wk config values from JS and scala?

Reply:
`/api/features/` exposes the `features` block of the config, maybe we can put it in there?